<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1064</width>
    <height>699</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(207, 209, 214);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" rowspan="2">
     <widget class="QGroupBox" name="groupBox">
      <property name="maximumSize">
       <size>
        <width>211</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(240, 240, 240);
</string>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>197</width>
         <height>241</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_10">
          <item>
           <widget class="QLabel" name="cool_temp_label">
            <property name="text">
             <string>Cool Temp *C</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="cool_temp_text">
            <property name="text">
             <string>-20</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_11">
          <item>
           <widget class="QLabel" name="heat_temp_label">
            <property name="text">
             <string>Heat Temp *C</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="heat_temp_text">
            <property name="text">
             <string>20</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_12">
          <item>
           <widget class="QLabel" name="heat_time_label">
            <property name="text">
             <string>Heat Time (sec)</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="heat_time_text">
            <property name="text">
             <string>60</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_13">
          <item>
           <widget class="QLabel" name="mode_label">
            <property name="text">
             <string>Mode</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="mode_select">
            <item>
             <property name="text">
              <string>Time</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Cycles</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Infinite</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_14">
          <item>
           <widget class="QLabel" name="run_time_label">
            <property name="text">
             <string>Run Time (min)</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="run_time_text">
            <property name="text">
             <string>1000</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_20">
          <item>
           <widget class="QLabel" name="export_file_name_label">
            <property name="text">
             <string>Export File Name</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="export_file_name_text">
            <property name="text">
             <string>trial</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_9">
          <item>
           <widget class="QPushButton" name="start_button">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(144, 214, 166);</string>
            </property>
            <property name="text">
             <string>Start Run</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="stop_button">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(255, 99, 99);</string>
            </property>
            <property name="text">
             <string>Stop Run</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="maximumSize">
       <size>
        <width>701</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QGroupBox" name="groupBox_4">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>211</width>
         <height>121</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(240, 240, 240);
</string>
       </property>
       <property name="title">
        <string/>
       </property>
       <widget class="QWidget" name="layoutWidget_3">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>191</width>
          <height>101</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="channel_1_label">
           <property name="styleSheet">
            <string notr="true">font: 10pt &quot;Inter&quot;;</string>
           </property>
           <property name="text">
            <string>Channel 1</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="channel_1_temp_label">
             <property name="text">
              <string>Temp:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="channel_1_temp_text">
             <property name="styleSheet">
              <string notr="true">background-color: rgb(255, 170, 127);
text-align: center;</string>
             </property>
             <property name="text">
              <string>N/A</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLabel" name="channel_1_voltage_label">
             <property name="text">
              <string>Voltage: </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="channel_1_voltage_text">
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>N/A</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="channel_1_current_label">
             <property name="text">
              <string>Current:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="channel_1_current_text">
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>N/A</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox_5">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>0</y>
         <width>221</width>
         <height>121</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(240, 240, 240);
</string>
       </property>
       <property name="title">
        <string/>
       </property>
       <widget class="QWidget" name="layoutWidget_2">
        <property name="geometry">
         <rect>
          <x>9</x>
          <y>9</y>
          <width>201</width>
          <height>101</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="channel_2_label">
           <property name="styleSheet">
            <string notr="true">font: 10pt &quot;Inter&quot;;</string>
           </property>
           <property name="text">
            <string>Channel 2</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QLabel" name="channel_2_temp_label">
             <property name="text">
              <string>Temp:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="channel_2_temp_text">
             <property name="styleSheet">
              <string notr="true">background-color: rgb(255, 170, 127);</string>
             </property>
             <property name="text">
              <string>N/A</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QLabel" name="channel_2_voltage_label">
             <property name="text">
              <string>Voltage: </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="channel_2_voltage_text">
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>N/A</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <widget class="QLabel" name="channel_2_current_label">
             <property name="text">
              <string>Current:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="channel_2_current_text">
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>N/A</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox_6">
       <property name="geometry">
        <rect>
         <x>450</x>
         <y>0</y>
         <width>251</width>
         <height>121</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(240, 240, 240);
</string>
       </property>
       <property name="title">
        <string/>
       </property>
       <widget class="QWidget" name="layoutWidget_4">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>231</width>
          <height>101</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLabel" name="status_label">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>50</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">font: 12pt &quot;Inter&quot;;</string>
           </property>
           <property name="text">
            <string>Device Status: OFF</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="connected_label">
           <property name="styleSheet">
            <string notr="true">font: 12pt &quot;Inter&quot;;</string>
           </property>
           <property name="text">
            <string>TEC Connected: NO</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="reset_button">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(209, 209, 209);</string>
           </property>
           <property name="text">
            <string>Reset TEC</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QWidget" name="graph_widget" native="true">
      <property name="maximumSize">
       <size>
        <width>701</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(240, 240, 240);
</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1064</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
